R/evalParsRollapply.R

```diff
@@ -34,8 +34,8 @@ evalParsTS <- function(par.matrix, object,
     }
   } else {
     ## Use disk
-    if (!require("parallel")) stop("package parallel is required for evalParsTS if filehash.name is not NULL and parallel$method is not 'foreach'")
-    if (!require("ff")) stop("package ff is required for evalParsTS if filehash.name is not NULL and parallel$method is not 'foreach'")
+    # if (!require("parallel")) stop("package parallel is required for evalParsTS if filehash.name is not NULL and parallel$method is not 'foreach'")
```

Comment:
It looks like removing this will cause the parallel package to cause an error elsewhere?
Perhaps it's better to keep both of these as requireNamespace?

Reply:
It didn't look like that, it seemed like you call everything parallel:: anyway later in the script.
More importantly should consider to include parallel in the dependencies? You use it in a quite a few scripts and might also open opportunities for making SCE a Dream parallel

Reply:
reinstated requireNamespace() in 4894467